_docs/payout-details.md

```diff
@@ -423,3 +423,114 @@ BJ
 <div class="alert alert-info" markdown="1">
 **Note** `XOF::Bank` payouts are currently in beta phase. At this time, we offer payouts to accounts in Senegal and Benin only.
 </div>
+
+# SouthAfrica
+
+## ZAR::Bank
+
+For South African bank payments please use:
+
+{% capture data-raw %}
+```javascript
+"details": {
+  "first_name": "First",
+  "last_name": "Last",
+  "street": "Main Street",
+  "postal_code": "AB0001",
+  "city": "Cape Town",
+  "bank_code": "334810",
+  "bank_account": "12345678",
+  "bank_name": "The Bank Name", // Not required if the bank is included in the list below
```

Comment:
I think we are using the bank_code alone now that we have the bank list.

Reply:
I have added this field just to be sure in case the bank is not included in the list (which according to what E4F support said it should be an extremely rare case)...should I remove it?

Reply:
I currently have it removed in api from the last test conversation. cc @sztupy

Reply:
let's remove it and supply the bank code list here - similar to how we do with NGN